website/hugo.yaml

```diff
@@ -76,13 +76,13 @@ params:
         url: "https://slack.k8s.io/"
         icon: fab fa-slack
         desc: "Chat with us on Slack in the #aws-provider channel"
-  latest_release_version: v0.34
+  latest_release_version: v0.35.0
```

Comment:
Looks like there may be something slightly wrong with the value that we're setting here now